src/main/java/org/folio/roles/service/capability/ApiRoleCapabilitySetService.java

```diff
@@ -24,6 +25,12 @@ public PageResult<RoleCapabilitySet> create(UUID roleId, List<UUID> capabilitySe
     return delegate.create(roleId, capabilitySetIds, safeCreate);
   }
 
+  @Override
+  public PageResult<RoleCapabilitySet> create(RoleCapabilitySetsRequest roleCapabilitiesRequest, boolean safeCreate) {
```

Comment:
it is roleCapabilitySetsRequest, but lets use just request to simplify as in ApiRoleCapabilityService